R/stats_methods.R

```diff
@@ -67,24 +67,25 @@ confint.fh <- function(object, parm = NULL, level = 0.95, ...) {
   }
 }
 
-# Extract the AIC from a model fit of an emdi object ---------------------------
+# Extract the AIC from a Model Fit of an emdi Object ---------------------------
 #' @export
 #' @method extractAIC fh
 #' @importFrom stats extractAIC
 
 extractAIC.fh <- function(fit, ...) {
   throw_class_error(fit, "fh")
   if (!is.null(fit$model$model_select$AIC)) {
-    message(paste0(
-      "Estimation approach used is ", fit$method$method, ": ",
-      round(fit$model$model_select$AIC, 5)
-    ))
+    message(strwrap(prefix = " ", initial = "",
+                    paste0("Estimation approach used is ",
+                           fit$method$method, ": ",
+                           round(fit$model$model_select$AIC, 5)
+                           )))
     invisible(fit$model$model_select$AIC)
   } else {
-    message(paste0(
-      "No AIC is returned for estimation approach ",
-      fit$method$method, "."
-    ))
+    message(strwrap(prefix = " ", initial = "",
+                    paste0("No AIC is returned for estimation approach ",
+                           fit$method$method, "."
+                           )))
```

Comment:
```suggestion
                    sprintf("No AIC is returned for estimation approach %s.", fit$method$method)
                           ))
```
Here, sprintf could be used as well (likewise in the following, similar cases), given that this is thought to be beneficial for readability :)